react-graphql-apollo-tutorial.md

```diff
@@ -1405,29 +1401,27 @@ const updateAddStar = (
 };
 {{< /highlight >}}
 
-Let's recap all three steps here. First, you have retrieved (read) the repository entity from the Apollo Client by using an identifier and the fragment. Second, you updated the desired information of the entity. And third, you wrote back the data whereas you include the updated information but keep all the remaining information intact which can be achieved by using the JavaScript spread operator. That's it for the whole manual update mechanism which you can do yourself when Apollo Client doesn't take care of it, because the mutation result hasn't all necessary data.
-
-After all, it is a best practice to have the identical fragment for all three parts: the initial query, the `readFragment()` and `writeFragment()` cache method. It is because your data structure for the particular entity stays consistent in your cache and there a not various definitions of this entity. For instance, if you forget to include a property which is defined by the fragment's fields in data object of the `writeFragment()` method, you will get a warning: *Missing field __typename in ...*. All of this helps you to embrace best practices when using Apollo Client.
+Let's recap all three steps here. First, you have retrieved (read) the repository entity from the Apollo Client using an identifier and the fragment; second, you updated the information of the entity; and third, you wrote back the data with updated information, but kept all remaining information intac using the JavaScript spread operator. This is a manual update mechanism that can be used when a mutation is missing data.
```

Comment:
"information intac using the JavaScript spread operator" to "information intact using the JavaScript spread operator"